src/python/Makefile

```diff
@@ -92,22 +91,22 @@ doc-build-test: install-tox
 	tox -e docs
 
 install-dep: check-root
-	pip install -r requirements.txt
+	pip3 install -r requirements.txt --user
 
 install: install-dep
-	pip install -e .
+	pip3 install -e . --user
```

Comment:
can use `python3 setup.py install --user` 

```Installing foglamp script to /home/foglamp/.local/bin

Installed /home/foglamp/.local/lib/python3.5/site-packages/FogLAMP-0.1-py3.5.egg
Processing dependencies for FogLAMP==0.1
Finished processing dependencies for FogLAMP==0.1
```
may be we also want to define v 0.0.1 per http://semver.org/